api/addons/urls.py

```diff
@@ -1,12 +1,12 @@
-from django.conf.urls import url
+from django.conf.urls import re_path
```

Comment:
Should we do the same for the admin app?

Reply:
Added in https://github.com/CenterForOpenScience/osf.io/pull/10070/commits/cb4264b99518cf2d764a06b65acb5ea5f8aa7dd5. In addition, applied the same to new API endpoints introduced during Keen Replacement.